sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -737,6 +737,9 @@ async def _client_ready_async(self):
             return False
         if self._link.get_state().value != 3:  # ATTACHED
             return False
+        # once the receiver client is ready/connection established, we set prefetch
+        if not self._link_credit:   # if prefetch = 0, set link_credit to 1 to keep receiving
+            self._link.link_credit = self._link_credit + 1
```

Comment:
as you mentioned in the other comment, I can change it so that link_credit=0 is passed in to create_receiver_link always. Then, at the SDK level, we explicitly send out a flow frame to "prefetch" right after the client is ready.

Reply:
setting the +1 in the pyamqp/uamqp transports.